frontend/src/component/changeRequest/ChangeRequest/SegmentTooltipLink.tsx

```diff
@@ -42,8 +42,9 @@ export const SegmentDiff: FC<{
     );
 };
 interface IStrategyTooltipLinkProps {
-    change: IChangeRequestUpdateSegment | IChangeRequestDeleteSegment;
```

Comment:
we were only reading current name from the change so removing [stamp coupling](https://wiki.c2.com/?StampCoupling) to the entire change

Reply:
Not sure I got much from the article you linked (it felt like it was arguing with itself too much), but sure, I don't mind this change.